.github/workflows/ci.yml

```diff
@@ -23,15 +18,26 @@ jobs:
           go-version: 1.18
       - uses: actions/checkout@v3
       - name: Building docker containers using docker-compose
-        run: GO_MOD=go.mod docker-compose up -d --build
+        run: GO_MOD=go.mod docker compose --env-file=./config/ci.env.dev up --wait --build backend db staging_db
       - name: Golang Tests
-        run: go test -v ./...
-        working-directory: ./backend
+        run: cd backend && go test -v ./...
+      - name: GetLogs
+        run: docker container logs go_backend
+
   frontend_testing:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
-      - name: type check
+      - name: Typescript TSC
         run: cd frontend && npm install && npm run types
-      - name: react tests
-        run: cd frontend && npm test
+      - name: React tests
+        run: |
+          GO_MOD=go.mod docker compose --env-file=./config/ci.env.dev up --wait --build backend db staging_db frontend
+      - name: Backend Logs
+        run: |
+          docker logs go_backend
+          docker logs pg_container
+          docker ps
+          docker logs go_backend
+#         docker exec frontend curl http://backend:8080/api/filesystem/info
+# docker exec frontend npm test
```

Comment:
Uh sorry so just to clarify but how is the docker compose build part running any sort of tests? And should should the lines at the bottom be commented out?

cc @sachk

Reply:
smh i pointed this out and now jared is STEALING the credit - more ABUSE by csesoc execs :(

Reply:
![ketchup-arma3](https://user-images.githubusercontent.com/48422312/203337212-afb92282-d8cc-4eb2-b1b9-eee8266252eb.gif)

Reply:
The bottom two lines were commented for testing reasons, the entire PR is still blocked by the weird networking issue 😢